packages/generator/src/service-name-formatter.spec.ts

```diff
@@ -112,6 +112,7 @@ describe('name-formatter', () => {
       expect(formatter.originalToComplexTypeName('BigNumber')).toBe(
         'BigNumber_1'
       );
+      expect(formatter.originalToEntityClassName('Service')).toBe('Service_1');
```

Comment:
Just a general comment. It is always nice to have tests not to test different things so that when a test fails you directly see what was broken. In this case it is still fine but often it is good to create a second dedicated `it` step and not put the expect in an existing test.

Reply:
I see. Thanks for sharing :)